includes/definitions/discovery/f5os.yaml

```diff
@@ -124,60 +124,70 @@ modules:
                     num_oid: .1.3.6.1.4.1.12276.1.2.1.2.2.1.4.8.112.108.97.116.102.111.114.109.7.110.118.109.101.48.110.49
                     descr: Total IOPS
                     group: Disk
-                    index: 0
+                    rrd_type: COUNTER
+                    index: diskTotalIops0
                 -
                     oid: F5-PLATFORM-STATS-MIB::diskUtilizationStatsTable
                     value: F5-PLATFORM-STATS-MIB::diskReadIops
                     num_oid: .1.3.6.1.4.1.12276.1.2.1.2.2.1.5.8.112.108.97.116.102.111.114.109.7.110.118.109.101.48.110.49
                     descr: Read IOPS
                     group: Disk
-                    index: 1
+                    rrd_type: COUNTER
+                    index: diskReadIops1
```

Comment:
Why the number after index?

Reply:
There is no real reason for that indeed. Only copy-paste with right click in VI, no 'd' :D I can fix that.

Reply:
done